### ISSUE TITLE
Multiple relay server issues

### ISSUE THREAD

**Issue by chonglee123456** [netbirdio/netbird#930]
Hello, may I ask if Netbird can set up or support multiple relay servers, and how to set them up? Thank you.

**Comment by mlsmaycon** [comment]
@chonglee123456 would you join our Slack and reach out so we can discuss this request in detail? and your use case.

**Comment by vampywiz17** [comment]
@mlsmaycon Yes, it would be good to install own relay server.

**Comment by mlsmaycon** [comment]
@vampywiz17 as an option for NetBird's cloud services?

**Comment by vampywiz17** [comment]
@mlsmaycon 

Yes. Now i use free plan and it is enugh to me (1 user 20 machine) but would be good that the direct conection is not possible, use a own coturn server. It faster and not burdening your infrastructure.

I know, that the self hosted management contain coturn, but i not want to host all management, only the relay.

**Comment by mlsmaycon** [comment]
Thanks for the feedback @vampywiz17 we will consider it for the next 2 quarters. Do you mind opening a separate feature request issue for your request?

**Comment by vampywiz17** [comment]
@mlsmaycon 

Done

https://github.com/netbirdio/netbird/issues/957
